.github/actions/torus-builder/entrypoint.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/sh
 
+set -e
+set -x
+
 RELEASE_SHA=$1
 
 mix local.hex --force
 mix local.rebar --force
-mix archive.install hex phx_new 1.5.9
+# mix archive.install hex phx_new 1.5.9 --force
```

Comment:
Talking with Eli, this line was no longer needed, and it was making local testing of the docker image difficult.